Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>maybe it'd be worth adding that these energies are for different sizes hence the difference? by the way, maybe we can run the full tutorial for the 3x3 directly, or is it too bad for symmetries? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Yeah, I may as well run a 3x3 to check against ED and something larger (say 6x6). The 3x3 lattice does have an extra factor of 2 of automorphisms, but as long as the ground state is symmetric w.r.t. that extra symmetry op, the model should still work. If anyone knows where the factor of two comes from let me know.